egs/libricss/SURT/dprnn_zipformer/beam_search.py

```diff
@@ -0,0 +1,1280 @@
+# Copyright    2021  Xiaomi Corp.        (authors: Fangjun Kuang
```

Comment:
Yeah I already replaced the ones that could be replaced. The others differ slightly from the original files. For example, in the `beam_search.py`, I use `SURT` model instead of `Transducer` with corresponding changes in the decoding methods.

Reply:
>  in the beam_search.py, I use SURT model instead of Transducer with corresponding changes in the decoding methods.

In the latest master, we are using `nn.Module` to replace `Transducer`, so it should be fine to replace it with a symlink.

Reply:
Ahh, I see. Thanks for pointing out.

I have replaced some other scripts with symlinks. For the particular case of beam_search.py, there are some small changes in the implementations, and I also plan to add SURT-specific decoding methods in the near future, so I think it may be okay to keep it separate for now.

Reply:
@csukuangfj do you think we can merge this soon?